includes/menu.inc

```diff
@@ -18,36 +18,33 @@ function oe_bootstrap_theme_preprocess_menu(&$variables, $hook) {
   $items = $variables['items'];
   foreach ($items as $key => $item) {
     $class = $item['in_active_trail'] ? ['active'] : [];
-    $menu_item = [
-      'label' => $item['title'],
-      'path' => $item['url']->toString(),
-      'attributes' => new Attribute(['class' => $class]),
-    ];
-    $variables['items'][$key] = $menu_item;
+    $variables['items'][$key]['label'] = $item['title'];
+    $variables['items'][$key]['path'] = $item['url']->toString();
+    $variables['items'][$key]['attributes'] = new Attribute(['class' => $class]);
 
     if (empty($item['below'])) {
       continue;
     }
 
     $class[] = 'nav-link';
-    $menu_item['attributes'] = new Attribute(['class' => $class]);
     $item_with_submenu = [
       'standalone' => TRUE,
       'dropdown' => TRUE,
       'link' => TRUE,
       'alignment' => 'dropdown-menu-sm-start',
-      'trigger' => $menu_item,
+      'trigger' => [
+        'label' => $item['title'],
+        'path' => $item['url']->toString(),
+        'attributes' => new Attribute(['class' => $class]),
+      ],
```

Comment:
@escuriola I see some repetition here vs above.
And I wonder, where is this 'trigger' value used, and which array keys does it expect?
In the `menu.html.twig`, I see we are directly calling the pattern.
In `pattern-navigation.html.twig` we are sending the items directly to `bcl-navigation.html.twig`.

Reply:
I see now where the 'trigger' key is used.
Imo this is a bit weird: By setting `$pattern_item['trigger']`, some other keys within `$pattern_item` such as `$pattern_item['label']` no longer have any effect.
Trying to clean this up would be out of scope in this issue. So let's keep it as it is.